packages/abstract-core/src/native/version_control.rs

```diff
@@ -119,6 +124,13 @@ pub enum ExecuteMsg {
         namespace: Namespace,
         monetization: Monetization,
     },
+    /// Sets the metadata configuration for a module.
+    /// This is an optional field for a module and default to ""
```

Comment:
What is an optional field? which field? 

This should mention that using the `Latest` variant sets the default metadata while specific versions overwrite that.